src/mysqlsh_helpers.py

```diff
@@ -57,6 +67,34 @@ class MySQLServiceNotRunningError(Exception):
     pass
 
 
+class MySQLConfirmAllInstancesOnlineError(Exception):
```

Comment:
`is_instance_configured_for_innodb` seems to not use it.

Reply:
great catch! this was added with some other code that is since deleted. removed in 5b30da9c233a0000a12b0c3dfb1a9da7dd6327f2